extensions/build.gradle

```
@@ -1,3 +1,3 @@
 ext {
-    groupIdentifier = "com.${organization}.extensions"
+    groupIdentifier = "com.${organization}.$projectIdentifier"
```

tie zatvorky sa mozu aj odstranit, ked sa to uz meni

```suggestion
    groupIdentifier = "com.$organization.$projectIdentifier"
```